deployment/environment/crib/ccip_deployer.go

```diff
@@ -93,8 +94,41 @@ func DeployCCIPAndAddLanes(ctx context.Context, lggr logger.Logger, envConfig de
 		})
 	}
 
-	// This will not apply any proposals because we pass nil to testing.
-	// However, setup is ok because we only need to deploy the contracts and distribute job specs
+	// set up chains
+	var addChainChangesets []commonchangeset.ChangesetApplication
+	chainConfigs := make(map[uint64]changeset.ChainConfig)
+	nodeInfo, err := deployment.NodeInfo(e.NodeIDs, e.Offchain)
+	if err != nil {
+		return DeployCCIPOutput{}, fmt.Errorf("failed to get node info from env: %w", err)
+	}
+	for _, chain := range chainSelectors {
+		chainConfigs[chain] = changeset.ChainConfig{
+			Readers: nodeInfo.NonBootstraps().PeerIDs(),
+			FChain:  1,
+			EncodableChainConfig: chainconfig.ChainConfig{
+				GasPriceDeviationPPB:    cciptypes.BigInt{Int: big.NewInt(1000)},
+				DAGasPriceDeviationPPB:  cciptypes.BigInt{Int: big.NewInt(1_000_000)},
+				OptimisticConfirmations: 1,
+			},
+		}
+	}
+
+	for range chainSelectors {
+		addChainChangesets = append(addChainChangesets, commonchangeset.ChangesetApplication{
+			Changeset: commonchangeset.WrapChangeSet(changeset.UpdateChainConfig),
+			Config: changeset.UpdateChainConfigConfig{
+				HomeChainSelector: homeChainSel,
+				RemoteChainAdds:   chainConfigs,
+			},
+		})
+	}
```

Comment:
you just need to apply it once. is there a need to iterate it for all chainSelectors?